remote-api/src/test/java/org/alfresco/rest/api/impl/rules/RulesImplTest.java

```diff
@@ -102,6 +102,13 @@ public void setUp() throws Exception
     {
         MockitoAnnotations.openMocks(this);
 
+        // Create the node validator.
+        NodeValidator validator = new NodeValidator();
+        validator.setNodes(nodesMock);
+        validator.setPermissionService(permissionServiceMock);
```

Comment:
Mock `permissionServiceMock` shouldn't be present here. All checks against this mock should be done in `NodeValidator` unit tests.

Reply:
I have fixed the `RulesImplTest` and moved some tests to new `NodeValidatorTest`.